experimental/packages/otlp-transformer/src/trace/internal.ts

```diff
@@ -73,6 +73,6 @@ export function toOtlpSpanEvent(timedEvent: TimedEvent): IEvent {
       : [],
     name: timedEvent.name,
     timeUnixNano: hrTimeToNanoseconds(timedEvent.time),
-    droppedAttributesCount: 0,
+    droppedAttributesCount: timedEvent.droppedAttributesCount || 0,
```

Comment:
```suggestion
    droppedAttributesCount: timedEvent.droppedAttributesCount,
```
Because it's no longer optional. Maybe there are similar places.